irods/message/__init__.py

```diff
@@ -232,7 +238,7 @@ class BinBytesBuf(Message):
     buf = BinaryProperty()
 
 
-class GSIAuthMessage(Message):
+class PluginAuthMessage(Message):
```

Comment:
The AuthMessage previously used for GSI is instead quite generic and also valid for PAM and other plugins

So that I renamed it as AuthPluginMessage